pandas/core/dtypes/dtypes.py

```diff
@@ -856,22 +856,15 @@ class PeriodDtype(PeriodDtypeBase, PandasExtensionDtype):
     _match = re.compile(r"(P|p)eriod\[(?P<freq>.+)\]")
     _cache_dtypes: dict[str_type, PandasExtensionDtype] = {}
 
-    def __new__(cls, freq=None):
+    def __new__(cls, freq):
         """
         Parameters
         ----------
-        freq : frequency
+        freq : PeriodDtype, BaseOffset, or string
         """
         if isinstance(freq, PeriodDtype):
             return freq
 
-        elif freq is None:
-            # empty constructor for pickle compat
```

Comment:
Do you know if this comment is still relevant?

Reply:
my assumption is that since it isn't in the test suite it should be OK

Reply:
Looks like this may break pre-0.19 pickle compat but that's fine IMO https://github.com/pandas-dev/pandas/commit/6fa2b03d78a8a6e94de66a01852a01a5574af2a9